### ISSUE TITLE
Save / restore window position after close KeeWeb desktop

### ISSUE THREAD

**Issue by darkkain** [antelle/keeweb#82]
Windows 10 (x64), Desktop Client v0.6.1
```
KeeWeb v0.6.1 (f6aa0c3, 2016-02-02)
Environment: electron v0.36.4
User-Agent: Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) KeeWeb/0.6.1 Chrome/47.0.2526.73 Electron/0.36.4 Safari/537.36
```
This issue about Electron "things". 
After close the app it's doesn't save window position and open with default settings, wich are specified in [app.js](https://github.com/antelle/keeweb/blob/master/electron/app.js#L83)

---
I can't offer correct PR, but can little bit search: I found out that electron don't have such API. But we have [npm-module](https://github.com/mawie81/electron-window-state) and [several](https://www.bountysource.com/issues/23689099-electron-window-state)  [snippets](https://github.com/atom/electron/issues/526)
Hope it's usefull for solve this "problem". 

Thanks for great product, u awesome :+1:  :smiley:

**Comment by antelle** [comment]
Done in 01e6533419a0a8caeb345dc3200de39a90756cd4, will be available in v1.1 release.

**Comment by darkkain** [comment]
Cool, thank you
and again: you awesome :smile:
